### PR TITLE
Fix module attr error to use python `__name__` instead of rust struct `name`

### PR DESCRIPTION
**Pull request by youknowone**

### CODE REVIEW

File: vm/src/pyobject.rs [hunk 1]
```diff
@@ -85,9 +84,6 @@ impl fmt::Display for PyObject<dyn PyObjectPayload> {
             }
         }
 
-        if let Some(PyModule { ref name, .. }) = self.payload::<PyModule>() {
```
Comment (youknowone) [inline vm/src/pyobject.rs]: is there any code depending on here? it is not possible to keep compatibility here without name field.

**Comment by coolreader18** [comment]
Why not just have a r/w property that gets/modifies the `name` struct field?

**Comment by youknowone** [comment]
How about delete in that case? I think it is not compatible with CPython.

File: vm/src/pyobject.rs (hunk 1 above)
Comment (windelbouwman) [reply vm/src/pyobject.rs]: We should use payload as little as possible outside of the object itself, so this removal seems good to me!

Comment (youknowone) [reply vm/src/pyobject.rs]: I confirmed current test code doesn't hit this line

**Pull request merged**